biom/table.py

```diff
@@ -2935,6 +2921,161 @@ def _intersect_id_order(self, a, b):
                 idx += 1
         return new_order
 
+    def concat(self, others, axis='sample'):
```

Comment:
i stepped back from wanting to make this a classmethod as, while not ideal, it would be inconsistent with the rest of the API